src/main/drivers/pwm_output_dshot_shared.c

```diff
@@ -194,21 +196,21 @@ static uint32_t decodeTelemetryPacket(uint32_t buffer[], uint32_t count)
     csum = csum ^ (csum >> 4); // xor nibbles
 
     if ((csum & 0xf) != 0xf) {
-        setDirectionMicros = micros() - start;
```

Comment:
The code may be styled as
```
do {
  if ((csum & 0xf) != 0xf) {
     ret = 0xffff;
     break;
 }

....
} while(0);
decodePacketDurationUs = micros() - start;
return ret

Reply:
The focus of this PR is the code dealing with the change from output to input via the dma handler.  This file was only touched because of the re-use of `setDirectioinMicros` that was used in the dma handler.

I'll point @joelucid at this comment but will leave that for another PR to keep this one foccussed and easy to review.